src/boost_histogram/__init__.py

```diff
@@ -24,9 +24,25 @@
 try:
     from . import _core
 except ImportError as err:
-    if "_core" in err.msg and "boost_histogram" in err.msg:
```

Comment:
MyPy discovered this error - "msg" is not public API and has a different name in Python 2.